working/augmentation-libraries/feature-specification.md

```diff
@@ -721,16 +729,85 @@ It is a compile-time error if:
 *   The constructor augmentation specifies any default values. *Default values
     are defined solely by the original constructor.*
 
-*   The original constructor is `const` and the augmenting constructor is not
+*   The augmented constructor is `const` and the augmenting constructor is not
     or vice versa.
 
-*   The original constructor is a factory constructor and the augmenting
-    constructor has an initializer list.
+*   The augmented constructor is a factory constructor and the augmenting
+    constructor is not or vice versa.
+
+*   The resulting constructor is not valid (has a redirecting initializer and
+    other initializers, multiple `super` initializers, etc).
+
+#### Non-redirecting generative constructors
+
+These are probably the most complex constructor, but also the most common.
+
+A non-redirecting generative constructor marked `augment` may:
+
+*   Add or replace the body of the existing constructor with a new body.
+
+    *   If the augmenting constructor has a body (a block or => body, not just
+        a single ;), then that body replaces the existing constructor body.
+
+    *   In the augmenting constructor's body, an `augmented()` call executes the
+        original constructor's body in the same parameter scope that the
+        augmenting body is executing in. The return type of this call is `void`.
+
+    *   Initializer lists _are not_ re-ran, they have already executed and
+        shouldn't be executed twice.
+
+    *   If a parameter variable is overwritten prior to calling `augmented()`,
+        the augmented body will see the updated value, because the parameter
+        scope is identical.
+
+    *   Local variables from the augmenting scope are not available in the
+        augmented scope.
+
+*   Add initializers to the initializer list.
+
+    *   These are appended to the original constructor's initializers, but before
+        any super initializer (if present).
+
+    *   They are allowed to add a super initializer, if none is present. It must
```

Comment:
Unclear what "They" refers to. (The initializers of the augmenting constructor's initializer list, I guess, but that's not a concept that's clear and close enough to refer to with an unqualifed "they").

If not using the above (which includes super-initializers), then maybe:
> The augmenting constructor's initializer list can contain a super-initializer, 
> which can be appended to the resulting initializer list if it doesn't already contain 
> a super-initializer.

Reply:
This is covered by the new wording you suggested